src/PipelinesMicroserviceCLI/Commands/ApprovePipelineRelease.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace PipelinesMicroserviceCLI\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use PipelinesMicroservice\Services\PipelineApi;
+use GuzzleHttp\Client;
+use PipelinesMicroservice\PipelinesMicroserviceApi;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class ApprovePipelineRelease extends PipelineManagerAPICommand
+{
+    protected $commandName = 'pipeline:approve';
+    
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $baseUrl  = $input->getArgument('base_url');
+        $api      = $this->getPipelineMicroserviceApi($baseUrl,$this->httpHandler);
+        $publishedPipelines = $api->pipelines->getPublished();
+        
+        if( !empty($publishedPipelines) ){
+            $messages    = [];
```

Comment:
Can we isolate logic more in methods so that the intention of the code is easier to read?